app/integrations/storidge/_module.js

```diff
@@ -1,5 +1,4 @@
-// TODO: legacy extension management
```

Comment:
I'd keep the legacy extension management note here as it is something that I want to change at some point (there are other occurences of this TODO in the codebase, will be useful for a future refactor).